### PR TITLE
optimize：Cj subsequent optimization

### PR DESCRIPTION
**Pull request by STATICHIT**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Background and solution
后续优化

### What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

## Summary by CodeRabbit

- **New Features**
	- Enhanced plugin navigation panel with drag-and-drop functionality for top and bottom navigation items.
	- Introduced a toggle feature for an AI robot component to improve user interaction with AI features.

- **Improvements**
	- Updated confirmation logic for switching plugins to enhance user experience.
	- Refined context menu logic for more dynamic interactions.
	- Added visual indicators for drag state in the navigation, canvas, and settings components.
	- Improved interactivity by integrating drag bar state into component rendering logic.

- **Refactor**
	- Improved overall structure and state management of the navigation lists for better organization and maintainability.

### CODE REVIEW

**Review by lizhijie429 — APPROVED** [review]

**Comment by hexqi** [comment]
image
插件隐藏功能没有生效

File: packages/common/component/PluginPanel.vue
Comment (hexqi) [inline packages/common/component/PluginPanel.vue:1]: 拖拽调整插件位置后刷新，插件位置会恢复到初始位置

Comment (hexqi) [inline packages/common/component/PluginPanel.vue:1]: image
拖拽时会选中界面文字

Comment (hexqi) [inline packages/common/component/PluginPanel.vue:1]: image
左侧插件拖拽到右侧后，宽度拖动条还是显示在右侧

**Review by hexqi — APPROVED** [review]

**Pull request merged**